apps/admin/backend/src/app.writeins.test.ts

```diff
@@ -122,177 +125,6 @@ test('getWriteInAdjudicationQueueMetadata', async () => {
   ]);
 });
 
-test('adjudicateWriteIn', async () => {
```

Comment:
Deleted this big test in favor of a more comprehensive big test below that I added.